docs/responses.md

```diff
@@ -182,6 +182,29 @@ async def app(scope, receive, send):
     await response(scope, receive, send)
 ```
 
+### EmptyResponse
+
+Supplies an empty message body as the response.  
+
+In particular, use an EmptyResponse object for 1xx, 204, 205, 304 responses as it sets or omits a Content-Length header as appropriate.
+
+Signature: `Response(status_code:int, headers: typing.Optional[typing.Dict[str, str]] = None, background: typing.Optional[BackgroundTask] = None)`
```

Comment:
```suggestion
Signature: `EmptyResponse(status_code:int, headers: typing.Optional[typing.Dict[str, str]] = None, background: typing.Optional[BackgroundTask] = None)`
```

Alternatively, just take this line out. We don't have proper, generated API docs so trying to be consistent is difficult.

Reply:
I fixed the class name.  Following the superclass documentation seemed like the right thing to do.